### ISSUE TITLE
Best way to parse libraryfolders.vdf without using dynamic?

### ISSUE THREAD

**Issue by DomiStyle** [GerhardvanZyl/Steam-VDF-Parser#2]
I'm trying to parse the libraryfolders.vdf with this library but unfortunately the format is a pain compared to the examples.

Here's an example of what the file looks like:

```
"libraryfolders"
{
	"contentstatsid"		"-1882263624787241560"
	"0"
	{
		"path"		"/path/to/steam/library"
		"label"		""
		"contentid"		"-1882263624787241560"
		"totalsize"		"0"
		"update_clean_bytes_tally"		"26553169288"
		"time_last_update_corruption"		"0"
		"apps"
		{
			"228980"		"940416973"
			"250820"		"5823064401"
			"858280"		"375543351"
			"961940"		"399675883"
			"1391110"		"639290775"
			"1580130"		"1107619726"
		}
	}
        "1"
	{
		"path"		"/path/to/another/steam/library"
		"label"		""
		"contentid"		"6490088256208813928"
		"totalsize"		"983429038080"
		"update_clean_bytes_tally"		"14876393334"
		"time_last_update_corruption"		"0"
		"apps"
		{
			"240"		"4585213376"
			"620"		"12718399062"
			"629"		"118928983"
		}
	}
}
```

Unfortunately `contentstatsid` ruins the idea of defining `libraryfolders` as a Dictionary. I could remove the entire line with contentstatsid since I don't need it but that feels more like a workaround.

Is there currently a way to ignore contentstatsid and still parse this as Dictionary without using dynamic?

**Comment by GerhardvanZyl** [comment]
Thanks for raising the issue. I released a new version 1.0.3 to address it.

If you want to just ignore contentstatsid, you can instantiate the deserializer with VdfDeserializer(true), which will ignore dictionary "entries" of the incorrect type.
Alternatively, you can create a derived class from Dictionary that includes the contentstatsid property.

```
    public class LibraryFolders<TKey, TValue> : Dictionary<TKey, TValue>
    {
        public string contentstatsid { get; set; }
    }

    public class Library
    {
        public LibraryFolders<int, LibraryFolder> libraryfolders { get; set; }
    }
```

I also fixed using ints as dictionary keys.

**Comment by DomiStyle** [comment]
Thanks for the quick fix!

Tested it just now and setting VdfDeserializer(true) works perfectly fine, the file parses as expected now.
